app/routes/preference.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../data')
```

Comment:
There are linting errors in this file which cause the tests to fail.

Make sure that you've got `standard` installed so it gives you fast feedback on these as you write code.  Also means you can turn on "fix on save" so to fix everything you just need to click save or do `CTRL+S`

You can install it by running `npm i -g standard` and it will install it globally on your machine so every repo can take advantage of it.  `-g` is the global bit, otherwise it would add it to the `package.json` of the repo only (which it should be there anyway)